server/src/controllers/appointments.controller.js

```diff
@@ -42,17 +35,7 @@ export async function newAppointment(req, res, next) {
       });
     }
 
-    const geocodeData = await geocodeAddress(address);
-
-    if (!geocodeData.results.length) {
```

Comment:
Well there's no longer such thing as an address without lat/long. We could error if the address is not found, which we arguably should've already been doing since these are the ones we consider valid. We're only enforcing that on the front end